chains/evm/chain.go

```diff
@@ -55,6 +55,11 @@ func SetupDefaultEVMChain(rawConfig map[string]interface{}, txFabric calls.TxFab
 	t := signAndSend.NewSignAndSendTransactor(txFabric, gasPricer, client)
 	bridgeContract := bridge.NewBridgeContract(client, common.HexToAddress(config.Bridge), t)
 
+	isRegiteredRelayer, _ := bridgeContract.IsRelayer(common.HexToAddress(rawConfig["from"].(string)))
```

Comment:
For it makes more sense to use `config.GeneralChainConfig.From` instead of `rawConfig["from"].(string)`

Reply:
> For it makes more sense to use `config.GeneralChainConfig.From` instead of `rawConfig["from"].(string)`

Somehow i missed that :see_no_evil: